recipes/lazperf/bld.bat

```diff
@@ -0,0 +1,15 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+      -DCMAKE_BUILD_TYPE:STRING=Release ^
+      -DWITH_TESTS:BOOL=OFF ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
Can we run the tests here too?

Reply:
Sure, although Windows is disabled for now, but better to do it know before forgetting.